cli/tests/activate.bats

```diff
@@ -3336,9 +3336,12 @@ EOF
 
 # bats test_tags=activate,activate:attach
 @test "attach doesn't break MANPATH" {
-  # We don't need an environment, but we do need wait_for_watchdogs to have a
-  # PROJECT_DIR to look for
-  project_setup_common
+  project_setup
+
+  # Ensure that an empty MANPATH is replaced with something with a trailing colon.
+  MANPATH= run "$FLOX_BIN" activate -- sh -c 'echo $MANPATH'
+  assert_success
+  assert_output --regexp ".*:$"
```

Comment:
(non-blocking, nit) This is already quite a large test that depends on incremental changes in state and now creates a third path environment that's unused in the later assertions. Splitting it out to a separate test could be clearer.

Reply:
I don't mind either way, but splitting it out in different tests would still involve creating three environments so the same work is performed either way? That, and logically grouping the MANPATH-related tests is in line with the similar adjacent test for PATH.

I could have also had the test use one of the vim or emacs environments, but figured that would be arbitrary while using an empty, rendered environment was the lightest test possible that wouldn't add to the cycles required for the test. I put the lightest test first so that if it failed then we could avoid the more expensive tests requiring vim and emacs.

Reply:
Chatted with Dan offline .. resolving conversation.